.eslintrc

```diff
@@ -14,7 +14,8 @@
     },
     "rules": {
         "prettier/prettier": ["error", { "doubleQuote": true }],
-        "no-console": 1
+        "no-console": 1,
+        "@typescript-eslint/explicit-function-return-type": "off"
```

Comment:
I'm just not sure about this config change, could you provide some motivation please? Personally I would rather have this on - are there some particularly troublesome functions?

Reply:
I do appreciate that it's a lot of work, I've started down this rabbit hole before 😆

Reply:
Hello, I would like to suggest because not all functions have return type.
It shows up as an eslint warning in VS Code.

If you don't want to specify a return type for every function, I think this rule will be useful to avoid confusion in your code.

Reply:
I would rather have all the function return types specified, but that can be done separately.